remme/rpc_api/event/_handlers.py

```diff
@@ -280,13 +299,24 @@ def parse_evt(self, evt):
             pass
 
     def validate(self, msg_id, params):
+
         from_block = params.get('from_block')
+        swap_id = params.get('id')
+
         if from_block and not isinstance(from_block, str):
-            raise ClientException(message='Invalid "from_block" type',
-                                  msg_id=id)
+            raise ClientException(message='Invalid "from_block" type', msg_id=id)
```

Comment:
`from_block` param must be also validated

Reply:
@DeV1doR, it is not scope of the ticket. @eugene-babichenko, add validation for `from_block` also in this pull request?

Reply:
@eugene-babichenko, done.